hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml

```diff
@@ -55,6 +55,24 @@
                 export KUBE_GCE_ZONE="us-central1-f"
                 export KUBEMARK_MASTER_SIZE="n1-standard-1"
                 export KUBEMARK_NUM_NODES="5"
+        - '100-gce':
+            description: 'Run small-ish kubemark cluster to continously run performance experiments'
+            timeout: 160
+            cron-string: 'H H/2 * * *'
```

Comment:
Do we really need to run it every 2 hours?
How about 3 times a day (or maybe 4?).